### PR TITLE
fix doc typos in several sections

### PR DESCRIPTION
**Pull request by tnielens**

### CODE REVIEW

File: docs/reference/stream/zsink.md [hunk 1]
```diff
@@ -295,9 +295,9 @@ val stream: ZIO[Any, Nothing, Int] =
 // Output: 10
 ```
 
-### From Effect
+### From ZIO
 
-The `ZSink.fromEffect` creates a single-value sink produced from an effect:
+The `ZSink.fromZIO` creates a single-value sink produced from an effect:
```
Comment (tnielens) [inline docs/reference/stream/zsink.md]: I wonder whether "effect" (used end of line here) is still in common use in the doc after these methods renames.

Comment (adamgfraser) [reply docs/reference/stream/zsink.md]: I might use "ZIO workflow" here.

**Review by adamgfraser — DISMISSED** [review]
Looks great! One minor suggestion.

File: docs/reference/stream/zsink.md (hunk 1 above)
Comment (tnielens) [reply docs/reference/stream/zsink.md]: I'll apply the suggestion.

**Review by adamgfraser — APPROVED** [review]

**Pull request merged**